test/scripts/backup-restore/j05-verify-3scale-postgres-backup-and-restore.sh

```diff
@@ -15,14 +25,19 @@ else
 fi
 
 # Set the parameters
+# Use passed in AWS credentials for STS, otherwise default to getting from cluster for normal OSD. Exit if these credentials are empty
+export AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID:=$(oc get secret aws-creds -n kube-system -o jsonpath='{.data.aws_access_key_id}' | base64 --decode)}
```

Comment:
it's possible that someone could be running this in a terminal session where they've already exported some credentials and forgotten? Would it be worth adding a step that the user should unset those in the instructions in the test case?